addons/website_event_meet/views/event_meet_templates_page.xml

```diff
@@ -30,7 +30,7 @@
 <template id="meeting_room_main" name="Meeting Room: Main Content">
     <div t-att-class="'col-12 o_wevent_online_page_main o_wemeet_room_main bg-white p-0 %s' % ('col-md-9 col-lg-10' if option_widescreen else 'col-md-8 col-lg-9')">
         <!-- EVENT NOT STARTED ALERTS -->
-        <div t-if="not meeting_room.event_id.is_ongoing" class="alert alert-warning text-center">
+        <div t-if="not meeting_room.event_id.is_ongoing and meeting_room.event_id.date_end &gt; datetime.datetime.now()" class="alert alert-warning text-center">
```

Comment:
event.is_ongoing = date_begin_utc <= now_utc <= date_end_utc

Is your new condition really necessary in that case ?

Reply:
Moreover you will have timezone issues.

Reply:
@tde-banana-odoo Yes, to not display the message if the event is finished

Reply:
But it is already included in event_id.is_ongoing computation normally. That's what I meant.

Reply:
Oh indeed, and I have the field `is_done` defined in event online which I can use

Reply:
(this field care about the timezone)

Reply:
Still not sure it is necessary, if you look at fields definition

            event.is_ongoing = date_begin_utc <= now_utc <= date_end_utc
            event.is_done = now_utc > date_end_utc

If event is done, normally is_ongoing is Falsy, unless I am mistaken.

Reply:
@tde-banana-odoo  Yes, if the event is not ongoing, it might be done or not